SPEC.md

```diff
@@ -5053,19 +5057,21 @@ workflow name {
 
 ### Workflow Elements
 
-Tasks and workflows have several elements in common. These sections have nearly the same usage in workflows as they do in tasks, so we just link to their earlier descriptions.
+Tasks and workflows have several elements in common. When applicable, the task definition for these sections is linked to rather than duplicated. 
 
-* [`input` section](#task-inputs)
-* [Private declarations](#private-declarations)
-* [`output` section](#task-outputs)
-* [`meta` section](#metadata-sections)
-* [`parameter_meta` section](#parameter-metadata-section)
+A workflow is comprised of the following elements:
 
-In addition to these sections, a workflow may have any of the following elements that are specific to workflows:
+* A singular, optional [`input`](#task-inputs) section (_identfical to the `input` section within tasks_).
+* A singular, optional [`output`](#task-outputs) section (_identfical to the `output` section within tasks_).
+* A singular, optional [`meta`](#metadata-sections) section (_identfical to the `meta` section within tasks_).
+* A singular, optional [`parameter_meta`](#parameter-metadata-section) section (_identfical to the `parameter_meta` section within tasks_).
+* Any number of workflow execution elements, which include the following:
```

Comment:
This introduces the term "workflow execution elements", which I would propose is what we call these execution-based elements. I considered using the term "workflow execution statements", but I just stuck with "elements" to be consistent with the language I used for tasks.